src/routes/docs/builders/tags-input/(sections)/Description.svelte

```diff
@@ -0,0 +1,351 @@
+<script lang="ts">
+	import { Docs } from '$routes/(components)';
+</script>
+
+<Docs.H2>Builder Description</Docs.H2>
+
+<Docs.P class="mt-4">
+	The following is an in-depth description of the return values from
+	<Docs.Code>createTagsInput()</Docs.Code>.
+</Docs.P>
+
+<Docs.H3><u>Stores</u></Docs.H3>
```

Comment:
We should avoid adding underline to H3, to keep a consistent style.

Reply:
Agreed. Sorry